.travis.yml

```diff
@@ -17,18 +17,26 @@ matrix:
       env:
         - OS=ubuntu-14.04
         - JOBQUEUE=sge
+    - python: "2.7"
+      env:
+        #For launching only basic unit tests
```

Comment:
About comments: I don't know about this one. Maybe there could be a comment that explains that "JOBQUEUE=none is for tests that do not need a cluster to run". Maybe also put it just before JOBQUEUE=none to make it more obvious that it is about the next line. Alternatively we could have find a better naming than `none` but I don't find any off the top of my head ...

Python bias related nitpick: I would put a space after the `#` ...

Reply:
Finaly, a semi-useful one 😁 !